features/login/forgot_password.feature

```diff
@@ -4,11 +4,54 @@ Feature: customer who forgot password can receive a new one by email
   So that I can login
   I want to generate a new password by email
 
-Scenario: reset password and login with new one
+Scenario: send magic link to user
+Given it is currently 'January 1, 2019 12:00'
+  Given customer "John Doe" exists and was created by admin
+  When I seed with 1
```

Comment:
This shouldn't be a Cucumber step.  The concept of seeding has no customer-facing/UI-facing counterpart.  For these tests, it may be better to just hardcode the token value for testing purposes.  I assume you have a private function called `generate_token` or something like that; for the test, you can stub that function and return a fixed known token value.